lambdas/nino-check/src/helpers/function-config.ts

```diff
@@ -0,0 +1,49 @@
+import { OtgConfig } from "../hmrc-apis/types/otg";
+import { PdvConfig } from "../hmrc-apis/types/pdv";
+import { AuditConfig, HmrcEnvVars, TableNames } from "../types/input";
+
+const envVarNames = {
+  sessionTable: "SESSION_TABLE",
+  personIdentityTable: "PERSON_IDENTITY_TABLE",
+  attemptTable: "ATTEMPT_TABLE",
+  ninoUserTable: "NINO_USER_TABLE",
+  auditEventBus: "AUDIT_EVENT_BUS",
+  auditSource: "AUDIT_SOURCE",
+  auditIssuer: "AUDIT_ISSUER",
+  pdvUserAgentParamName: "PDV_USER_AGENT_PARAM_NAME",
+};
+
+export class NinoCheckFunctionConfig {
+  public readonly tableNames: TableNames;
+  public readonly audit: AuditConfig;
+  public readonly hmrcApi: HmrcEnvVars;
+
+  constructor({ deviceInformationHeader }: { deviceInformationHeader?: string }) {
```

Comment:
I'm interested to see `deviceInformationHeader` come in here just to be appended to the `audit` object. 

What's the reason that we can't use it directly? It's not really lambda config it's information about a users specific device - [docs are here](https://github.com/govuk-one-login/architecture/blob/7182b823f79d6edcc5a6cd11733ce2904c3bc204/rfc/0023-crossteam-audit-and-analysis-eventmessage-payload.md#device-information-object)

Reply:
Yeah, fair question - I'm happy to pull it out. I think the reason I did this was originally because I was having trouble with function parameter counts, so I decided to include it in the function config. I guess in that case it would have really been state rather than config. The current version of the handler has much fewer issues with function parameter counts, so I'm happy to just pass it into the audit functions directly.

Reply:
Ok yes please, let's pass directly